File: 프로그래머스/unrated/161990. 바탕화면 정리/바탕화면 정리.java
class Solution {
    public int[] solution(String[] wallpaper) {
        int[] answer = new int[4];
        
        int lux = wallpaper.length;
        int luy = wallpaper[0].length();
        int rdx = 0;
        int rdy = 0;
        
        for(int i = 0; i < wallpaper.length; i++){
            String s = wallpaper[i];
            if(!s.contains("#")){
                continue;
            }
            
            int x = i;
            int y = s.indexOf("#");
            
            lux = Math.min(x,lux);
            luy = Math.min(y,luy);
            rdx = Math.max(x,rdx);
            rdy = Math.max(y,rdy);
            
            y = s.lastIndexOf("#");
            rdx = Math.max(x,rdx);
            rdy = Math.max(y,rdy);
            
        }
        
        answer[0] = lux;
        answer[1] = luy;
        answer[2] = ++rdx;
        answer[3] = ++rdy;
        
        
        return answer;
    }
}

File: 프로그래머스/unrated/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    answer = []
    a, b = [], []
    
    #wallpaper 내부 전체 돌면서
    for i in range(len(wallpaper)): 
        for j in range(len(wallpaper[0])):
            if(wallpaper[i][j]=="#"):
                a.append(i)
                b.append(j)
    #와
    return (min(a), min(b), max(a)+1, max(b)+1)